### ISSUE TITLE
Linting issue

### ISSUE THREAD

**Issue by embabi2005** [python-LimeReport/python-LimeReport#8]
Linting is working with PySide6 v6.4 on Ubuntu but not on windows. With PySide6 v6.7, linting isn't working on both windows and ubuntu. I use vscode.

**Comment by Maks1mS** [comment]
Hi @embabi2005,

Thank you for raising this issue!

To help me better understand the problem, could you please provide more details? Specifically:
What do you mean by "linking isn't working"? Are you seeing specific error messages, or is linting simply not being triggered at all? Can you provide a screenshot?

This additional information would be really helpful in diagnosing the issue. Thanks!

**Comment by embabi2005** [comment]
First Thank you for the effort you do for this valuable project that helps me a lot.

Second, I mean the code drop down suggestion and completion as shown in the below screenshots. Though, the code executes normally. 

Sorry, I mixed up between Linting and IntelliSense.

Update:
I tried PyCharm and IntelliSense is working.

![Screenshot 2024-10-29 180414](https://github.com/user-attachments/assets/e4ed4816-34f9-4900-8e16-c9d6b623e496)

![Screenshot 2024-10-29 180359](https://github.com/user-attachments/assets/1f015abd-e2a2-4ab6-ad77-c43ae564ffa8)
